test/login.e2e-spec.ts

```diff
@@ -0,0 +1,95 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication, ValidationPipe } from '@nestjs/common';
+import * as request from 'supertest';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import * as dotenv from 'dotenv';
```

Comment:
nit: wonder if this can be used in root app

Reply:
I placed files and modified them to work in the root folder, but could not find a way to apply them to the test environment.